heroku_connect/models.py

```diff
@@ -102,9 +101,8 @@ class TriggerLogAbstract(models.Model):
                              choices=TRIGGER_LOG_STATE_CHOICES)
     sf_message = models.TextField(editable=False, null=True, blank=True)
 
-    # HStoreFields need 'django.contrib.postgres' in INSTALLED_APPS and hstore postgres extension
-    values = HStoreField(editable=False, null=True, blank=True)
-    old = HStoreField(editable=False, null=True, blank=True)
+    values = models.TextField(editable=False, null=True, blank=True)
+    old = models.TextField(editable=False, null=True, blank=True)
```

Comment:
Does Django generate a migration for such case?
So, for example if we introduce this change to systems already using old code, would that work without an issue?

Reply:
it's a breaking change on the library level

Reply:
(but no migrations, since it is an unmanaged model)

Reply:
Ah, true, right!
Then we would need to cover this in the release notes.